test/controllers/MakingALateAppealControllerSpec.scala

```diff
@@ -162,6 +164,13 @@ class MakingALateAppealControllerSpec extends SpecBase {
         val result: String = controller.getHeadingAndTitle()(userRequestWithCorrectKeys, implicitly)
         result shouldBe "This penalty was issued more than 30 days ago"
       }
+      "the user has selected maybe to appealing both penalties" in new Setup(AuthTestModels.successfulAuthResult) {
```

Comment:
This gives the impression that 'maybe' is a valid option. Perhaps rename this to something like:
```suggestion
      "the user has an unknown answer to appealing both penalties" in new Setup(AuthTestModels.successfulAuthResult) {
```